person.rb

```diff
@@ -2,14 +2,15 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rents
 
   def initialize(age, parent_permission: true, name: 'Unknown')
     super()
     @id = Random.rand(0..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rents = []
   end
 
```

Comment:
- [ ] In your `person.rb`, kindly add a method that creates a new instance of the `Rental` class. Remember the relationship between `Book` and `Person` is a many-many relationship and they are joined through `Rentals`. This means that for a `Book` to connect with the `Person`, a `Book` should have many `Rentals` and a `Person` should have many `Rentals`. Kindly create this method that creates a new instance of Rental in the `Person` class. 

#### _**Hint:**_ The method should have parameters for an instance of `Book`, and the `date`. Then when you create the instance of `Rental`, you can pass `self`. 👍

Reply:
Roger!